server/app.js

```diff
@@ -19,11 +20,14 @@ mongoose.connection.on('error', function(err) {
 });
 
 // Setup server
-var app = express();
-var server = http.createServer(app);
+const app = express();
+const server = http.createServer(app);
 require('./config/express').default(app);
 require('./routes').default(app);
 
+//using cookie-parser
+app.use(cookieParser());
```

Comment:
All middlewares are registered in express.js file. This is not right place to register middlewares